docs/modules/ROOT/pages/how-tos/vcluster.adoc

```diff
@@ -0,0 +1,125 @@
+= Lieutenant in vcluster
+
+Vcluster is a virtual Kubernetes Clusters that run inside regular namespaces.
+Installing Lieutenant in a vcluster can be helpful as it allows you to run multiple versions of Lieutenant, with different CRD versions on the same cluster.
+
+The following steps need to be performed to deploy Lieutenant in a vcluster.
+
+== Deploy vcluster and enable Syn
+
+First we need a vcluster instance we can deploy Lieutenant to.
+Follow the setup guide in the https://hub.syn.tools/vcluster/index.html[component-vcluster documentation] and be sure to enable Syn on the newly created virtual cluster.
+
+After that you should be able to deploy to a vcluster and the vcluster component configuration should look similar to the following:
+
+[code,yaml]
+----
+applications:
+  - vcluster as lieutenant-prod
+parameters:
+  lieutenant_prod:
+    host_service_cidr: 172.30.0.0/16
+    namespace: lieutenant-prod
+    ocp_route:
+      host: lieutenant-prod.apps.cluster.example.com
```

Comment:
This seems quite OCP-specific. The docs should assume plain K8s for components which are hosted on github.com/projectsyn.